packages/perseus/src/widgets/interactive-graphs/graphs/components/use-control-point.tsx

```diff
@@ -72,6 +75,12 @@ export function useControlPoint(params: Params): Return {
             className="movable-point__focusable-handle"
             tabIndex={disableKeyboardInteraction ? -1 : 0}
             ref={focusableHandleRef}
+            role="button"
+            aria-label={strings.srPointAtCoordinates({
+                x: String(point[X]),
+                y: String(point[Y]),
+            })}
+            aria-live="assertive"
```

Comment:
Did you add `aria-live="assertive"` for the unlimited points graph? This wouldn't do anything for a finite points graph, right?

Reply:
It might also be good to add a comment here about why we're using "assertive" instead of "polite"

Reply:
`aria-live="assertive"` causes the new location of the point to be announced whenever you move it. I'll add a comment!